src/plugins/orangehrmTimePlugin/Api/TimesheetCommentAPI.php

```diff
@@ -188,6 +218,31 @@ public function getValidationRuleForDelete(): ParamRuleCollection
     }
 
     /**
+     * @OA\Get(
+     *     path="/api/v2/time/timesheets/{timesheetId}/entries/{id}/comment",
+     *     tags={"Time/Timesheet Comment"},
+     *     @OA\PathParameter(
+     *         name="timesheetId",
+     *         @OA\Schema(type="integer")
+     *     ),
+     *     @OA\PathParameter(
+     *         name="id",
+     *         @OA\Schema(type="integer")
+     *     ),
+     *     @OA\Response(
+     *         response="200",
+     *         description="Success",
+     *         @OA\JsonContent(
+     *             @OA\Property(
+     *                 property="data",
+     *                 ref="#/components/schemas/Time-TimesheetItemModel"
+     *             ),
+     *             @OA\Property(property="meta", type="object")
+     *         ),
+     *         @OA\Response(response="404", ref="#/components/responses/RecordNotFound")
+     *     )
```

Comment:
response inside  another response
```suggestion
     *         )
     *     ),
     *     @OA\Response(response="404", ref="#/components/responses/RecordNotFound")
```